redbot/cogs/trivia/data/lists/2016.yaml

```
@@ -1,4 +1,5 @@
 AUTHOR: JennJenn
+DESCRIPTION: A general knowledge trivia based around the year of 2016
```

Missing full stop, "year of" doesn't seem to be correct grammar:
https://english.stackexchange.com/questions/306301/in-the-year-2016-in-the-year-of-2016-of-the-year-2016
https://english.stackexchange.com/questions/8788/in-the-year-1908-or-in-the-year-of-1908
```suggestion
DESCRIPTION: A general knowledge trivia based around the year 2016.
```